src/thunks/getMoviesThunk.test.js

```diff
@@ -0,0 +1,68 @@
+import {
+  fetchDefaultData
```

Comment:
Should we change this naming to reflect that ti doesn't just fetch default data?